OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -539,13 +543,22 @@ private void initDatesBanner(CourseBannerInfoModel courseBannerInfo) {
         }
     }
 
+    private void detectDeepLinking() {
```

Comment:
This method will open the first component of the course as the `courseComponentId` is updated with first courseComponent before calling the deeplinking.
Please  move `courseComponentId = courseComponent.getId();` from `line:717` to `line:763` after calling this method

Reply:
done.